### ISSUE TITLE
Auto-correction of BracesAroundHashParameters with trailing commas breaks code

### ISSUE THREAD

**Issue by ryym** [rubocop-hq/rubocop#6175]
Hi,
I found a case that the auto-correction of the `Style/BracesAroundHashParameters` cop results in a syntax error.

`.rubocop.yml`:

```yaml
Style/TrailingCommaInHashLiteral:
  Enabled: true
  EnforcedStyleForMultiline: comma

Style/TrailingCommaInArguments:
  Enabled: true
  EnforcedStyleForMultiline: comma

Style/BracesAroundHashParameters:
  Enabled: true
```

`a.rb`:

```ruby
hello(
  :bob,
  {
    a: 1,
    b: 2,
  },
)
```

--------

## Expected behavior

RuboCop auto-corrects `a.rb` like this:

```ruby
hello(
  :bob,
  a: 1,
  b: 2,
)
```

## Actual behavior

RuboCop auto-crrects `a.rb` like this:

```ruby
hello(
  :bob,
    a: 1,
    b: 2,
  ,
)
```

The result is obviously an syntax error 😱 

```shell
$ ruby -c a.rb
a.rb:5: syntax error, unexpected ',', expecting ')'
```

## Steps to reproduce the problem

```shell
bundle exec rubocop -a a.rb
```

## RuboCop version

```
$ bundle exec rubocop -V
0.58.2 (using Parser 2.5.1.2, running on ruby 2.4.1 x86_64-darwin16)
```
